wire/protocol.go

```diff
@@ -183,6 +183,9 @@ const (
 	// TestNet3 represents the test network (version 3).
 	TestNet3 BitcoinNet = 0x0709110b
 
+	// SigNet represents the signet (default).
+	SigNet BitcoinNet = 0x40CF030A
```

Comment:
Is this documented anywhere? Doesn't seem to be in the OG BIP.

Reply:
I copied the constant from https://en.bitcoin.it/wiki/Protocol_documentation#Common_structures
The constant is equal to `chaincfg.SigNetParams.Net`, this is checked in new test TestSigNetMagic.

Reply:
It should also be equal to `CustomSignetParams(DefaultSignetChallenge, nil).Net`, or:

```
	challengeLength := byte(len(DefaultSignetChallenge))
	hashDouble := chainhash.DoubleHashB(
		append([]byte{challengeLength}, challenge...),
	)
	net := binary.LittleEndian.Uint32(hashDouble[0:4])
```

Perhaps we can add a simple test case for it?

Reply:
Ah, I guess that's what we're doing with `TestSigNetMagic`, so never mind.